app/code/core/Mage/Index/Model/Event.php

```diff
@@ -71,7 +71,7 @@ protected function _construct()
     /**
      * Specify process object
      *
-     * @param Mage_Index_Model_Process $process
+     * @param Mage_Index_Model_Process|null $process
```

Comment:
How about line 86? `* @return Mage_Index_Model_Process` to `* @return Mage_Index_Model_Process|null`?

Reply:
Thanks.